src/styles/app/46-quick-note.css

```
@@ -0,0 +1,121 @@
+		color-mix(in srgb, var(--interactive-accent) 42%, transparent);
+	outline-offset: 2px;
+}
+
+.quickNoteSaveButton:disabled {
+	opacity: 0.5;
+	pointer-events: none;
+}
+
+.quickNoteSaveButton > svg {
+	flex: 0 0 auto;
+	color: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)